membership-attribute-service/app/services/SupporterRatePlanToAttributesMapper.scala

```diff
@@ -250,7 +250,7 @@ object SupporterRatePlanToAttributesMapper {
     ) -> tierThreeTransformer,
     List(
       "71a1c43a1e192b28f702b3b47113000a",
```

Comment:
There's a new rate plan ID in code too.

Reply:
@david-pepper new Rate Plan ID for CODE: `71a1bebf6be9444afad446c5ebaf0019`

Reply:
Done